modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -59,6 +66,53 @@ public String getFilesystemFriendlyName() {
                 assertEquals("Recorded file not found", 1, files.length);
             }
         }
+
+        @Test
+        public void recordingTestThatShouldHaveCorrectDuration() throws IOException, InterruptedException {
+            File target = vncRecordingDirectory.getRoot();
+            try (
+                // recordAll {
+                // To do this, simply add extra parameters to the rule constructor:
+                BrowserWebDriverContainer<?> chrome = new BrowserWebDriverContainer<>()
+                    .withCapabilities(new ChromeOptions())
+                    .withRecordingMode(RECORD_ALL, target)
+                    .withRecordingFileFactory(new DefaultRecordingFileFactory())
+                    .withNetwork(NETWORK)
+            ) {
+                chrome.start();
+
+                TimeUnit.SECONDS.sleep(5);
```

Comment:
Why do you need this sleep?

Reply:
Sometimes the test goes very fast and the record may have **duration = 00:00:00.00**,  I reduce sleep to 1 second.